R/labs_to_levels.R

```diff
@@ -0,0 +1,116 @@
+#' @title Convert value labels into factor levels
+#' @name labs_to_levels
+#'
+#' @details
+#' Convert variables or data into factors and uses value labels as factor levels.
+#' This function is similar to `to_factor()`, however, it is specially designed
+#' for factors with value labels attributes. While `to_factor()` returns factors
+#' as is, `labs_to_levels()` always uses value labels as factor levels, even
```

Comment:
"specially designed for factors with value labels attributes" -> contradicts the NEWS bullet that says it converts data into factors